src/core/agent/services/connectionService.ts

```diff
@@ -123,6 +124,24 @@ class ConnectionService extends AgentService {
     return connectionRecord.state === DidExchangeState.Completed;
   }
 
+  async resolveOObi(url: string, name: string): Promise<void> {
```

Comment:
This a auxiliar method to resolve a a oobi url from the tunnel extension and store the response in the Preferences, to be used as:
`await AriesAgent.agent.connections.resolveOObi(oobiUrlValue, oobiNameValue);`